### PR TITLE
Improve scale out event with active trigger's name

### PR DESCRIPTION
**Pull request by SpiritZhou**
Add active trigger's name into scale up event for more clear information

### Checklist

- [X] Tests have been added
- [X] Changelog has been updated and is aligned with our [changelog requirements](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#Changelog)
- [x] A PR is opened to update the documentation on ([repo](https://github.com/kedacore/keda-docs)) *(if applicable)*
- [X] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))

Relates to https://github.com/kedacore/keda/issues/5577

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -63,6 +63,7 @@ Here is an overview of all new **experimental** features:
 
 ### Improvements
 
+- **General**: Add active trigger name in ScaledObject's scale up event ([#5577](https://github.com/kedacore/keda/issues/5577))
```
Comment (tomkerkhove) [inline CHANGELOG.md:66]:
```suggestion
- **General**: Add active trigger name in ScaledObject's scale out event ([#5577](https://github.com/kedacore/keda/issues/5577))
```

**Review by tomkerkhove — APPROVED** [review]

**Comment by tomkerkhove** [comment]
Can you update the docs as well please?

**Review by JorTurFer — COMMENTED** [review]
Looking good! 😄 
Should we update the status with the scalers which have triggered the scaling from 0?

**Comment by SpiritZhou** [comment]
> Looking good! 😄 Should we update the status with the scalers which have triggered the scaling from 0?

I can create another PR to update the status of ScaledObject with active trigger and keep this PR simple. WDYT？

File: pkg/scaling/executor/scale_scaledobjects.go [hunk 2]
```diff
@@ -309,7 +310,7 @@ func (e *scaleExecutor) scaleFromZeroOrIdle(ctx context.Context, logger logr.Log
 		logger.Info("Successfully updated ScaleTarget",
 			"Original Replicas Count", currentReplicas,
 			"New Replicas Count", replicas)
-		e.recorder.Eventf(scaledObject, corev1.EventTypeNormal, eventreason.KEDAScaleTargetActivated, "Scaled %s %s/%s from %d to %d", scaledObject.Status.ScaleTargetKind, scaledObject.Namespace, scaledObject.Spec.ScaleTargetRef.Name, currentReplicas, replicas)
+		e.recorder.Eventf(scaledObject, corev1.EventTypeNormal, eventreason.KEDAScaleTargetActivated, "Scaled %s %s/%s from %d to %d, triggered by %s", scaledObject.Status.ScaleTargetKind, scaledObject.Namespace, scaledObject.Spec.ScaleTargetRef.Name, currentReplicas, replicas, strings.Join(activeTriggers, "/")) // nosemgrep
```
Comment (JorTurFer) [inline pkg/scaling/executor/scale_scaledobjects.go:313]: I think that we can avoid the semgrep detection if we do something like:
```suggestion
		e.recorder.Eventf(scaledObject, corev1.EventTypeNormal, eventreason.KEDAScaleTargetActivated, "Scaled %s %s/%s from %d to %d, triggered by %s", scaledObject.Status.ScaleTargetKind, scaledObject.Namespace, scaledObject.Spec.ScaleTargetRef.Name, currentReplicas, replicas, strings.Join(activeTriggers, ";"))
```
Is it important for us using '/'?

**Comment by JorTurFer** [comment]
> I can create another PR to update the status of ScaledObject with active trigger and keep this PR simple. WDYT？

yeah, It's perfect

**Review by JorTurFer — APPROVED** [review]

**Pull request merged**